packages/contracts-bedrock/test/FaultDisputeGame.t.sol

```diff
@@ -139,7 +139,7 @@ contract FaultDisputeGame_Test is FaultDisputeGame_Init {
     ///               For now, it is critical that the first proposed output root of an OP stack
     ///               chain is done so by an honest party.
     function test_initialize_firstOutput_reverts() public {
-        uint256 submissionInterval = l2OutputOracle.submissionInterval();
```

Comment:
We shouldn't need to make this diff if the new abi is preserved

Reply:
you're correct, but I'm going to keep this diff because it's more consistent with the rest of this test file.